package.json

```diff
@@ -54,7 +54,7 @@
     "mongoose-paginate-v2": "^1.3.18",
     "morgan": "^1.10.0",
     "netmask": ">=2.0.1",
-    "pm2": "^4.5.5",
+    "pm2": "^5.1.2",
```

Comment:
Our work currently doesn't involve dealing with proxy configuration in-depth but Version above 5 is good to go as malicious requests on the local network to remotely run arbitrary code inside the Node.js process whenever an HTTP request is created is restricted.

Reply:
I didn't get this, Can you please elaborate on what I need to do to resolve this one.